src/transformState.js

```diff
@@ -5,7 +5,30 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  const changedState = state;
+
+  for (const action of actions) {
+    switch (action.type) {
+      case 'addProperties':
+        Object.assign(changedState, action.extraData);
+        continue;
```

Comment:
Interesting solution with continue, but I think it would be better to use break in such cases, and default to display an unknown command message.